### PR TITLE
New POM

### PR DESCRIPTION
**Pull request by jglick**
Supersedes #303.

- [X] use new parent POM
- [X] use split test harness
- [X] switch baseline to 1.609.3
- [x] fix `java.lang.IncompatibleClassChangeError: Implementing class` (cf. https://github.com/jenkinsci/jenkins-test-harness/pull/2)

@reviewbybees esp. @andresrc

### CODE REVIEW

File: aggregator/pom.xml [hunk 1]
```diff
@@ -35,6 +35,13 @@
     <name>Pipeline (formerly known as Workflow)</name>
     <url>https://wiki.jenkins-ci.org/display/JENKINS/Pipeline+Plugin</url>
     <dependencies>
+        <dependency> <!-- TODO pending https://github.com/jenkinsci/plugin-pom/pull/3 -->
```
Comment (amuniz) [inline aggregator/pom.xml]: Released as `plugin:2.1`

File: pom.xml [hunk 2]
```diff
@@ -28,7 +28,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>1.609.2</version> <!-- but see Jenkins-Version override below -->
+        <version>2.0</version>
```
Comment (amuniz) [inline pom.xml]: 2.1

**Comment by amuniz** [comment]
:bee:

**Comment by andresrc** [comment]
:bee:

I had to release `plugin:2.1` because of https://github.com/jenkinsci/plugin-pom/pull/3 and https://github.com/jenkinsci/plugin-pom/pull/5.

Sorry for the inconvenience.

**Comment by jglick** [comment]
The CI server seems to be down so I am running tests locally after the latest two commits.

**Comment by jglick** [comment]
Passed locally.

**Comment by amuniz** [comment]
:bee:

**Comment by andresrc** [comment]
:bee:

**Comment by amuniz** [comment]
:bee:

**Pull request merged**